vimrc

```diff
@@ -132,12 +133,6 @@ let g:html_indent_tags = 'li\|p'
 set splitbelow
 set splitright
 
-" Quicker window movement
-nnoremap <C-j> <C-w>j
-nnoremap <C-k> <C-w>k
-nnoremap <C-h> <C-w>h
-nnoremap <C-l> <C-w>l
```

Comment:
It was removed to make way for the https://github.com/christoomey/vim-tmux-navigator plugin instead. :smile: